study.md

```diff
@@ -66,7 +67,8 @@ Which ActiveRecord method finds all of the records of a certain type (or
 entity)?
 
 ```md
-<!-- your response here -->
+READ
+.where will allow you to find all records that match a criteria you give
```

Comment:
`#where` will return everything if no filters are passed into it.  `#all` will return all records of a certain type.